tests_root/wdio/features/home.feature

```diff
@@ -0,0 +1,12 @@
+@Home_Screen
+Feature: Home Screen of Beep Web App
+  As a User
+  I want to navigate to the appropriate screen
+  So that I can check if my account name or password have been compromised
+
+  @S1 @manual
```

Comment:
If this test is automated, please remove the @manual tag

Reply:
Oops! Good catch! 
Addressed in commit aa2beb4.